tools/ci_build/github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml

```diff
@@ -358,6 +358,7 @@ jobs:
         --skip_submodule_sync \
         --parallel \
         --use_nnapi \
+        --build_shared_lib \
         --cmake_generator=Ninja \
```

Comment:
Is this change needed? Wondering if there's any implication on the code coverage results if we start building a shared lib instead.

Reply:
Ouch, that might be a case, indeed. So I think I should move the test to `onnxruntime_test_all` target instead? So that we don't need the `--build_shared_lib` option

Reply:
I don't know for sure. As we're only plugging in the dummy demangle if the shared library is being built, moving the test has other implications. I'd try a temporary change in the CI config to comment out the `condition` so it runs code coverage for all builds to see if there is any impact on it.

Reply:
Elaborate?

> comment out the `condition`

Reply:
```yaml
# The below jobs only run on master build
- job: NNAPI_EP_MASTER
  pool:
    vmImage: 'macOS-11'
  timeoutInMinutes: 120
  workspace:
    clean: all
  condition: in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI')
```

Last line there determines if the job runs. Comment that out temporarily so it should run for all builds.